RocketChat/Rocket.Chat
# [FIX] Additional alias names of custom emojis not accepted anymore #18264

[FIX] Additional alias names of custom emojis not accepted anymore #18264


  - I have read the Contributing Guide - https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat doc
  - I have signed the CLA - https://cla-assistant.io/RocketChat/Rocket.Chat
  - Lint and unit tests pass locally with my changes
  - I have added tests that prove my fix is effective or that my feature works (if applicable)
  - I have added necessary documentation (if applicable)
  - Any dependent changes have been merged and published in downstream modules

## Proposed changes 

![Screenshot 2023-02-26 032734](https://user-images.githubusercontent.com/107949435/221405275-9cea4705-3c20-4044-864a-508fd0c267ac.png)






## Issue(s)

#18264 

## Steps to test or reproduce


1. Add a new custom emoji
2. Give it multiple aliases, e.g. first second
3. Hit save, see how the alias does not get transformed into "first_second" and instead they are separated with a space "first second"


## Further comments
